app/services/filtered_artwork_search_service.rb

```diff
@@ -1,9 +1,27 @@
 module FilteredArtworkSearchService
   class ServiceError < StandardError; end
 
+  ALLOWED_PARAMS = %i[
+    acquireable
+    artist_ids
+    attribution_class
+    fair_id
+    for_sale
+    gene_ids
+    keyword
+    offerable
+    partner_id
+    price_range
+    sort
+    tag_id
+  ].freeze
```

Comment:
This is motivated by Rails' paranoia about controller params. Having this list simplifies the parameter allowlisting over in `MatchController#filter_artworks`.